src/shogun/lib/type_case.h

```diff
@@ -464,6 +471,12 @@ static const typemap sg_non_integer_typemap = {
 		ADD_TYPE_TO_MAP(floatmax_t , TYPE::T_FLOATMAX)
 		ADD_TYPE_TO_MAP(complex128_t, TYPE::T_COMPLEX128)
 };
+static const typemap sg_object_typemap = {
+        ADD_TYPE_TO_MAP(CSGObject* , TYPE::T_SGOBJECT)
+        ADD_TYPE_TO_MAP(CKernel* , TYPE::T_CKERNEL)
+        ADD_TYPE_TO_MAP(CLikelihoodModel*, TYPE::T_CLIKELIHOOD_MODEL)
+        ADD_TYPE_TO_MAP(CMeanFunction*, TYPE::T_CMEANFUNCTION)
```

Comment:
@gf712 Here too the object typemaps stuff.

Reply:
yup, that's fine!